code/math/vecmat.cpp

```diff
@@ -470,9 +470,6 @@ float vm_vec_copy_normalize(vec3d *dest, const vec3d *src)
 
 	//	Mainly here to trap attempts to normalize a null vector.
 	if (m <= 0.0f) {
-		mprintf(("Null vec3d in vec3d normalize.\n"
```

Comment:
This one has helped when we have to debug object updates in multiplayer, so an nprintf for that one would be nice.

Reply:
There's no "Math" category, is "Network" OK?

Reply:
Yes, thanks